### PR TITLE
Feature/calculator functionality

### PR DESCRIPTION
**Pull request by matovu-farid**
The following changes were added in this branch:
- The logic to calculate and operate on the numbers was added.
- The handleButtonClick function was added to handle button clicks in the calculator and could be triggered when a button is clicked.
- The handleOpperate function was created to handle operations and is triggered when the equals button is clicked.
- The Answer component was configured to display the output of the calculation.

### CODE REVIEW

File: src/components/List/List.js [hunk 1]
```diff
@@ -10,13 +11,19 @@ class TypePad extends Component {
   }
 
   render() {
+    const { handleButtonClick, handleOpperate } = this.props;
     return (
       <ul className="flex">
-        <MainList />
-        <OperationList />
+        <MainList handleButtonClick={handleButtonClick} />
+        <OperationList handleButtonClick={handleButtonClick} handleOpperate={handleOpperate} />
```
Comment (Meltrust) [inline src/components/List/List.js:18]: - [ ] According to the project requirements:

image

Therefore, kindly let's implement the use of React lifecycle methods (you can keep it simple), so we can move closer to approval 💪

**Review by Meltrust — CHANGES_REQUESTED** [review]
Hi @matovu-farid, 


## You met and exceeded most of the project requirements!!! Great implementation!! 
There are some issues that you still need to work on to go to the next project but you are almost there!



## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Meltrust — APPROVED** [review]
Hi @matovu-farid,

## Great job!!! 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/List/List.js (hunk 1 above)
Comment (matovu-farid) [reply src/components/List/List.js:18]: I have added a lifecycle method in Calculator.js.

**Pull request merged**